### PR TITLE
Add webdriver test for pointer_move action within an iframe

### PR DESCRIPTION
**Pull request by whimboo**
Those added tests demonstrate that for both Chrome and Safari the pointer actions are misbehaving when executed within an iframe. The target coordinates as defined for the action aren't reached because the offset of the iframe isn't taken into account.

CC @sadym-chromium and @gsnedders

### CODE REVIEW

**Comment by whimboo** [comment]
@jgraham can you please review?

File: webdriver/tests/classic/perform_actions/pointer_mouse.py [hunk 1]
```diff
@@ -204,6 +204,53 @@ def test_move_to_position_in_viewport(
...
+        </script>
+    """
+        )
+    )
+
+    frame = session.find.css("iframe", all=False)
+    session.switch_frame(frame)
+
+    elem = session.find.css("input", all=False)
+    elem_center_point = list(center_point(elem))
```
Comment (juliandescottes) [inline webdriver/tests/classic/perform_actions/pointer_mouse.py:234]: question: do we really need to convert to a list here? We have a tuple and we never have to update it, so it should be good as is?

**Review by juliandescottes — APPROVED** [review]
LGTM, just one nit

File: webdriver/tests/classic/perform_actions/pointer_mouse.py (hunk 1 above)
Comment (whimboo) [reply webdriver/tests/classic/perform_actions/pointer_mouse.py:234]: No, we don't. Thank you for catching that. It's actually a relict from before a slight refactoring when I was comparing those values with an expected event build up as usual via a list. I'll remove.

**Comment by whimboo** [comment]
The Chromium related bug is: https://github.com/GoogleChromeLabs/chromium-bidi/issues/2630

**Pull request merged**